### PR TITLE
CDAP-7280 Update to verification information

### PR DESCRIPTION
**Pull request by awholegunch**
Update to the verification step information.

Fix for https://issues.cask.co/browse/CDAP-7280

Running as a Quick Build: http://builds.cask.co/browse/CDAP-DQB246-4 (passes)

Page of interest: http://builds.cask.co/artifact/CDAP-DQB246/shared/build-4/Docs-HTML/html/admin-manual/verification.html

### CODE REVIEW

File: cdap-docs/admin-manual/source/operations/index.rst [hunk 1]
```diff
@@ -81,29 +81,56 @@ Command Line Interface. See :ref:`reference:cli` in the
...
   
     http://<host>:10000/v3/system/services
 
+  Such as (for the Standalone CDAP):
+   
+  .. tabbed-parsed-literal::
+
+    $ curl -w"\n" -X GET "http://localhost:10000/v3/system/services"
+
   On success, the call returns a JSON string with component names and their corresponding
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/operations/index.rst]: this returns more than status, right?

File: cdap-docs/admin-manual/source/operations/index.rst [hunk 2]
```diff
@@ -123,3 +150,28 @@ On success, the calls return a valid HTTP response with a 200 code.
       ingestion.","status":"OK","logs":"OK","min":1,"max":1,"requested":1,"provisioned":1},
      {"name":"transaction","description":"Service that maintains transaction
       states.","status":"OK","logs":"NOTOK","min":1,"max":1,"requested":1,"provisioned":1}]
+
+- To retrieve a **health check of all the services running in YARN**, make a GET request
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/operations/index.rst]: ... all the CDAP system services?

File: cdap-docs/admin-manual/source/operations/index.rst [hunk 3]
```diff
@@ -123,3 +150,28 @@ On success, the calls return a valid HTTP response with a 200 code.
...
+  
+    http://<host>:10000/v3/system/services/status
+
+  Such as (for the Standalone CDAP):
+   
+  .. tabbed-parsed-literal::
+
+    $ curl -w"\n" -X GET "http://localhost:10000/v3/system/services/status"
+
+- To retrieve a **health check of a particular service running in YARN**, make a GET request
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/operations/index.rst]: ... a particular CDAP system service?

File: cdap-docs/admin-manual/source/operations/index.rst (hunk 1 above)
Comment (awholegunch) [reply cdap-docs/admin-manual/source/operations/index.rst]: Changed to add "and details of the services"

File: cdap-docs/admin-manual/source/operations/index.rst (hunk 2 above)
Comment (awholegunch) [reply cdap-docs/admin-manual/source/operations/index.rst]: Added.

File: cdap-docs/admin-manual/source/operations/index.rst [hunk 4]
```diff
@@ -81,31 +81,58 @@ Command Line Interface. See :ref:`reference:cli` in the
...
-On success, the calls return a valid HTTP response with a 200 code.
+  Such as (for the Standalone CDAP):
+   
+  .. tabbed-parsed-literal::
+
+    $ curl -w"\n" -X GET "http://localhost:10009/status"
 
-- To retrieve the **health check of all the services running in YARN**, make a GET request
-  to the URI::
+- To retrieve a **list of health checks of all the CDAP system services running in YARN**, make a GET request
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/operations/index.rst:123]: nit: is "running in YARN" important in this context? Nothing about the content suggests any YARN specific stuff. I may be thinking too far into the future, but we could have a release where we support Mesos and then we'd have to update this :)

**Review by bdmogal — APPROVED** [review]
Tiny nit. Other than that, LGTM

File: cdap-docs/admin-manual/source/operations/index.rst (hunk 4 above)
Comment (awholegunch) [reply cdap-docs/admin-manual/source/operations/index.rst:123]: I could be wrong, but I thought this was to distinguish services such as CDAP Master, which are run outside YARN, right? But perhaps we need further clarification?

Comment (bdmogal) [reply cdap-docs/admin-manual/source/operations/index.rst:123]: but does it make sense in this context? Any way, this is not a huge thing, we can keep it.

**Pull request merged**